test/advection_test/test_host.F90

```diff
@@ -460,28 +487,73 @@ subroutine test_host(retval, test_suites)
          write(6, '(a,i0,a,a,i0,/,a)') "ERROR: Error, ", errflg, " trying ",  &
               "to set thermo_active prop for cld_ice index = ", index_ice,    &
               trim(errmsg)
-         errflg_final = -1 !Notify test script that a failure occured
+         errflg_final = -1 !Notify test script that a failure occurred
       end if
       if (errflg == 0) then
          call const_props(index_ice)%is_thermo_active(check, errflg, errmsg)
          if (errflg /= 0) then
             write(6, '(a,i0,a,i0,/,a)') "ERROR: Error, ", errflg,             &
                  " tryingto get thermo_active prop for cld_ice index = ",     &
                  index_ice, trim(errmsg)
-            errflg_final = -1 !Notify test script that a failure occured
+            errflg_final = -1 !Notify test script that a failure occurred
          end if
       end if
       if (errflg == 0) then
          if (.not.check) then !Should now be True
             write(6, *) "ERROR: 'set_thermo_active' did not set",             &
                  " thermo_active constituent property correctly."
-            errflg_final = -1 !Notify test script that a failure occured
+            errflg_final = -1 !Notify test script that a failure occurred
          end if
       else
          !Reset error flag to continue testing other properties:
          errflg = 0
       end if
 
+      !Check that setting a constituent's default value works as expected
+      call const_props(index_liq)%has_default(has_default, errflg, errmsg)
+      if (errflg /= 0) then
+         write(6, '(a,i0,a,i0,/,a)') "ERROR: Error, ", errflg, " trying ", &
```

Comment:
You'll need another `a` in the format string because you technically now have two different output strings in a row:

```suggestion
         write(6, '(a,i0,2a,i0,/,a)') "ERROR: Error, ", errflg, " trying ", &
```

Reply:
fixed!